backend-tests/tests/test_tenantadm.py

```diff
@@ -92,8 +96,8 @@ def test_duplicate_username(self, clean_mongo_tenant_migration):
         )
         self.logger.debug("Tenant id: %s" % tenant_id)
 
-        # Retry login every second for 3 min
-        for i in range(60 * 3):
+        # Retry login every second for 2 min
+        for i in range(60 * 2):
```

Comment:
Sorry for the question: why are we reducing it from 3 mins to 2 mins?
What's the rationale? The commit message doesn't explain it.

Reply:
Saving user to the database does not take 3 minutes for sure, I'm trying to save some time in frequently running tests.